tests/__init__.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+
+
+class IndexSearchTestSupport:
```

Comment:
As we identified test data/functions used for both index and search tests, we wanted to put it in a common location, and (right or wrong) selected tests/__init__.py. At the time, putting it in IndexSearchTestSupport seemed a useful way to organize and access it. Yet, it would be simpler and more standard to just make them module variables/functions.
I will do that.